interfaces/builtin/system_observe.go

```diff
@@ -117,6 +117,16 @@ dbus (send)
     interface=org.freedesktop.DBus.Peer
     member=GetMachineId
     peer=(label=unconfined),
+
+# Allow reading if protected hardlinks are enabled, but don't allow enabling or
+# disabling them
+@{PROC}/sys/fs/protected_hardlinks r,
+@{PROC}/sys/fs/protected_symlinks r,
+@{PROC}/sys/fs/protected_fifos r,
+@{PROC}/sys/fs/protected_regular r,
+
+# Allow reading what size hugepages have been configured on a system.
+/sys/kernel/mm/hugepages/ r,
```

Comment:
note that these accesses are also required by Greengrass, but they don't need to go into the specific greengrass-support interface since greengrass also needs system-observe for the other interfaces, so I figured this was an appropriate place to put these accesses.